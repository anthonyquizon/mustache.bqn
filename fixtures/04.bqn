data⇐{x⇐⟨{y⇐{z⇐1}}⋄{y⇐{z⇐2}}⟩}
