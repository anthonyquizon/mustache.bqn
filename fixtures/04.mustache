<a>{{#x}}<b>{{#y}}<c>{{z}}</c>{{/y}}</b>{{/x}}</a>
