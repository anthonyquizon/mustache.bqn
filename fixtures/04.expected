<a><b><c>1</c></b><b><c>2</c></b></a>
